plutus-core/generators/Language/PlutusCore/Generators/NEAT/Spec.hs

```diff
@@ -126,21 +121,21 @@ not exploited.
 
 -- handle a user error and turn it back into an error term
 handleError :: Type TyName DefaultUni ()
-       -> ErrorWithCause (EvaluationError user internal) term
-       -> Either (ErrorWithCause (EvaluationError user internal) term)
+       -> U.ErrorWithCause (U.EvaluationError user internal) term
```

Comment:
has the error handling machinery moved into the untyped code? I guess that's fine.

Reply:
It's the same error handling machinery, just reexported from both typed and untyped CEK machines.